CHANGES.md

```diff
@@ -4,21 +4,24 @@
 
 This edge release updates the proxy-init container to check whether the iptables
 rules have already been added, which prevents errors if the proxy-init container
-is restarted. Also, the linkerd cli now has tab completion for kubernetes
-resources, saving you precious keystrokes! Finally, the proxy has been updated
-with several fixes and improvements.
+is restarted. Also, the `viz stat` command of the linkerd CLI now has tab
```

Comment:
Nit: I think users can assume we're talking about the Linkerd CLI 😬

I also think it's still worth noting that more completions are on the way!

```suggestion
is restarted. Also, the `viz stat` command now has tab
```